include/libcyphal/transport/udp/msg_rx_session.hpp

```diff
@@ -122,9 +123,20 @@ class MessageRxSession final : private IRxSessionDelegate, public IMessageRxSess
 
     // MARK: IRxSessionDelegate
 
-    void acceptRxTransfer(const UdpardRxTransfer&) override
+    void acceptRxTransfer(UdpardRxTransfer& inout_transfer) override
```

Comment:
This is non-`const` now deliberately - we move memory fragments from this RX transfer into the `UdpardMemory` instance mentioned previously. Hence parameter called as `inout_transfer`.